### PR TITLE
[RFC] Documented the installation process for Symfony Flex

### PR DESCRIPTION
**Pull request by sheeep**
This is an overhaul of the installation process documentation in the `README.md` file.
There is a few things this documentation relies on:

* [ ] `contao/core-bundle` v4.6 must be released so the [recipe PR](https://github.com/symfony/recipes-contrib/pull/426) can be merged.
* [x] The Installation bundle must support the `DATABASE_URL` environment variable configuration, so [PR #95](https://github.com/contao/installation-bundle/pull/95) must be merged

If you try this with a bare bone Symfony  (`composer create-project symfony/website-skeleton`) you will run run into an error, as the `json_manifest_file` does not yet exist.

Be sure to disable the manifest file in `config/assets.yaml`, as you won't be able to run `yarn run encore dev` without adding at least one asset file. But you'd need the manifest file in order to run the `asset()` calls in the error template of Contao.

I'm pretty sure this edge case is out of scope of this documentation, that's why I left it out.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,77 +14,45 @@ CMS functionality to any Symfony application. If you do not have an existing
...
-Then run `php composer.phar update` to install the vendor files.
+… or follow the instructions to use the contrib recipe in the installation process.
 
-Note that you can exchange the `php-http/guzzle6-adapter` package with any other [HTTP client implementation][4]. If you already have an HTTP client implementation, you can omit the package entirely.
+Add the `contao-component-dir` to the `extra` section of your `composer.json` file.
 
-## Activation
+```json
+"extra": {
+  "contao-component-dir": "assets"
```
Comment (leofeyer) [inline README.md]: Why not `composer config extra.contao-component-dir assets` similar to `extra.symfony.allow-contrib`?

File: README.md [hunk 2]
```diff
@@ -231,3 +174,5 @@ Visit the [support page][5] to learn about the available support options.
 [3]: https://github.com/contao/managed-edition
 [4]: https://packagist.org/providers/php-http/client-implementation
 [5]: https://contao.org/en/support.html
+[6]: https://github.com/symfony/recipes-contrib
+[7]: http://symfony.com/doc/current/components/dotenv.html
```
Comment (xchs) [inline README.md]: `https://symfony.com/doc/current/components/dotenv.html`

File: README.md [hunk 3]
```diff
@@ -94,7 +62,8 @@ ContaoCoreBundle:
     resource: "@ContaoCoreBundle/Resources/config/routing.yml"
 ```
 
-Edit your `app/config/security.yml` file:
+Edit your `app/config/security.yml` file and merge all the `providers`, `encoders`, `firewalls`
```
Comment (leofeyer) [inline README.md]: Should this be `config/security.yml` instead of `app/config/`?

**Comment by leofeyer** [comment]
Thank you @sheeep.

**Pull request merged**